frontend/src/hooks/useWindowDimensions.js

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
+// https://stackoverflow.com/questions/36862334/get-viewport-window-height-in-reactjs
+const getWindowDimensions = () => {
+  const { innerWidth: width, innerHeight: height } = window;
```

Comment:
not sure if it would work but, could this line be removed so that this function just returns
```jsx
{
  width: window.innerWidth,
  height: window.innerHeight,
}
```

Reply:
oh i realised i stopped using this file. Ill remove it.